db/column_family.cc

```diff
@@ -449,9 +449,7 @@ void SuperVersion::Cleanup() {
     to_delete.push_back(m);
   }
   current->Unref();
-  if (cfd->Unref()) {
-    delete cfd;
-  }
+  cfd->UnrefAndTryDelete(this);
```

Comment:
Just a question: when does `ColumnFamilyData` have different `SuperVersion`?
I thought they're always point to each other for the same CF.

Reply:
From new comment

    // (But while installing a new SuperVersion, this
    // cfd could be referenced only by two SuperVersions.)

Reply:
oh, I see, one is the new SV, another one is the old SV.